src/components/text-area/text-area.tsx

```diff
@@ -200,12 +194,12 @@ export class TextArea
   /**
    * Fires each time a new `value` is typed.
    */
-  @Event({ cancelable: true }) calciteTextareaInput: EventEmitter<void>;
+  @Event({ cancelable: true }) calciteTextAreaInput: EventEmitter<void>;
```

Comment:
I don't see any logic to deal with cancelation. If these are not cancelable, can you update the event decorator options?